### PR TITLE
Created github build workflow

### PR DESCRIPTION
**Pull request by icnocop**
Converted from Visual Studio 2008 to Visual Studio 2010
Update projects to target .NET Framework 4

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
```
Comment (dblock) [inline .github/workflows/build.yml:5]: Do it on all branches (*) and it should build this PR as well.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+    branches: [ main ]
```
Comment (dblock) [inline .github/workflows/build.yml:9]: We don't have main rn.

**Review by dblock — COMMENTED** [review]
Let's get a running CI on this pull request?

**Comment by icnocop** [comment]
I used `banches-ignore` to ignore the `downloads` branch but the pull request is not run in this repo, probably because the GitHub workflow is only in https://github.com/icnocop/vmwaretasks.
You can see the latest CI build [here](https://github.com/icnocop/vmwaretasks/actions/runs/1907315234).

**Comment by dblock** [comment]
I merged it. Make a PR that adds a badge and we should see it work.

**Pull request merged**